routes/encrypt.js

```diff
@@ -0,0 +1,12 @@
+const bcrypt = require('bcryptjs');
+
+
+exports.encrypt = (newUser) => new Promise((resolve, reject) => {
+  bcrypt.genSalt(10, (err, salt) => {
```

Comment:
how about this? do we have promises from box?

Reply:
yes